lighthouse-core/audits/accessibility/td-headers-attr.js

```diff
@@ -15,12 +15,12 @@ const AxeAudit = require('./axe-audit.js');
 const i18n = require('../../lib/i18n/i18n.js');
 
 const UIStrings = {
-  /** Title of an accesibility audit that evaluates if all table cell elements in a table that use the headers HTML attribute use it correctly to refer to cells within the same table. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  /** Title of an accesibility audit that evaluates if all table cell elements in a table that use the headers HTML attribute use it correctly to refer to header cells within the same table. This title is descriptive of the successful state and is shown to users when no user action is required. */
   title: 'Cells in a `<table>` element that use the `[headers]` attribute only refer ' +
-      'to other cells of that same table.',
-  /** Title of an accesibility audit that evaluates if all table cell elements in a table that use the headers HTML attribute use it correctly to refer to cells within the same table. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
+      'to headers of that same table.',
+  /** Title of an accesibility audit that evaluates if all table cell elements in a table that use the headers HTML attribute use it correctly to refer to header cells within the same table. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
   failureTitle: 'Cells in a `<table>` element that use the `[headers]` ' +
-      'attribute refers to other cells of that same table.',
+      'attribute refers to other cells of that same table, or elements outside the table.',
```

Comment:
WDYT about?

```suggestion
      'attribute do not refer to headers of the same table.',
```

either way we should fix the `refers` typo though :)

Reply:
Sure.  Tbh it's saying the same thing.  But this is more clear.

`refers to other cells` == `do no refer to headers`

Reply:
oh yeah just trying to avoid the `, or elements in another table` extra stuff, glad you like it! :)